tasks/queue.py

```diff
@@ -40,13 +40,19 @@ class MilMoveHHGQueueTasks(QueueTaskSet):
     def on_start(self):
         """ """
 
-    @task
-    def start_milmove_flow(self):
-        """
-        Start a flow and put it on the queue
-        """
+    @task(2)
```

Comment:
Would it be ok/appropriate to link to http://docs.locust.io/en/stable/writing-a-locustfile.html and explain what the `(2)` means and why we want that task to have a higher weight? I think it would be really helpful to others